bucket/tealdeer.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "1.5.0",
+    "description": "A very fast implementation of tldr in Rust.",
+    "homepage": "https://dbrgn.github.io/tealdeer",
+    "license": "Apache-2.0|MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/dbrgn/tealdeer/releases/download/v1.5.0/tealdeer-windows-x86_64-msvc.exe#/tldr.exe",
+            "hash": "399e5ddcf600b8ae7a9629990a2943360dec01bb47bf5d9f3b90e14ff4f88ede"
+        }
+    },
+    "pre_install": [
+        "$file = 'config.toml'",
+        "if (!(Test-Path \"$persist_dir\\$file\")) {",
+        "   Write-Host 'File' $file 'does not exists. Creating.' -f Yellow",
+        "   $CONT = @(",
+        "       '[directories]'",
+        "       \"custom_pages_dir = `\"$persist_dir\\custom-pages`\"\"",
+        "   ) -replace '\\\\', '\\\\'",
```

Comment:
Replaced `\\\\` with `\\\\`?

Reply:
Former is RegEx (a single `\`), latter is double `\`, since expression in "" treat `\\` as `\` and RegEx escape `\` with `\\`...

Reply:
Okay, understood